js/data/gateway-api/gatewayApi.js

```diff
@@ -169,7 +169,7 @@ export const hentSpinnsynBackendUrl = () => {
         return 'http://localhost:6932/api/v1/vedtak';
     } if (url.indexOf('localhost') > -1 || erNaisLabsDemo()) {
         // Lokalt
-        return `${process.env.REACT_APP_SYFOREST_ROOT}/vedtak`;
```

Comment:
Dette vil endre hvordan det funker her: 

https://sykefravaer.labs.nais.io/

Ser nå at det ikke funker helt optimalt der i dag heller. Det returneres ikke vedtak på det endepunktet den prøver å treffe. 

Stemmer for at den iffen deles i to en for localhost og en for naislabsdemo. Og så må du passe på at den treffer et mock endepunkt i naislabs versjonen. Helst uten /syforest siden dette ikke har noe med syforest å gjøre

Reply:
Ja, den jeg har fjernet pekte feil. Jeg fjernet "/syforest" fra mock endepunktet i forrige commit, men glemte å endre i gatewayApi :p 
Denne er riktig: http://localhost:8080/vedtak